definitions/tools/unaligned_seq_fda_stats.cwl

```diff
@@ -121,6 +117,22 @@ requirements:
                 sub Main {
                     # local variables
                     
+                    
+                    # gets the input file format
+                    unless (defined $format)
```

Comment:
It looks like this is being defined on line 72.  Will we ever fall into this case?  If we don't fall into this case for whatever reason, does that mean the user must NOT pass the format lest it be added to `@paths` instead?